src/Entities/Chore.php

```diff
@@ -4,8 +4,6 @@
 namespace Choredo\Entities;
 
 use Ramsey\Uuid\UuidInterface;
-use Doctrine\ORM\Mapping as ORM;
```

Comment:
Your IDE is probably complaining, but I really don't think those annotations work without this. There's a PHPStorm plugin for Doctrine annotations, if you're interested. https://plugins.jetbrains.com/plugin/7320-php-annotations